app/components/themes/create-record/component.coffee

```diff
@@ -0,0 +1,32 @@
+`import Ember from 'ember'`
+
+mergeArrayPromises = (promises) ->
+  promise = Ember.RSVP.all(promises).then((arrays) ->
+    mergedArray = Ember.A()
+    arrays.forEach((records) -> mergedArray.pushObjects(records.toArray()))
+    mergedArray
+  )
+  DS.PromiseArray.create({promise: promise})
+
+ThemesEditRecordComponent = Ember.Component.extend
+  store: Ember.inject.service()
+  actions:
+    search: (term) ->
+      return [] if Ember.isBlank(term)
+      mergeArrayPromises [
+        @theme.store.query('theme', {}),
```

Comment:
@madnificent I have an issue here with the backend. If I try to filter with `{filter: {prefLabel: term}}` there is a traceback in the backend. Let me know when you have time.

```
dispatcher_1  | 
dispatcher_1  | 10:29:26.185 [info]  GET /themes
dispatcher_1  | Forwarding request to http://resource/themes/?filter%5BprefLabel%5D=a
resource_1    | [2016-04-15 10:29:26 [ERROR]] Condition MU-CL-RESOURCES::NO-SUCH-LINK was signalled.
resource_1    | Backtrace for: #<SB-THREAD:THREAD "hunchentoot-worker-172.18.0.4:52220" RUNNING {10069ABC93}>
resource_1    | 0: ((LAMBDA NIL :IN SB-DEBUG::FUNCALL-WITH-DEBUG-IO-SYNTAX))
resource_1    | 1: (SB-IMPL::CALL-WITH-SANE-IO-SYNTAX #<CLOSURE (LAMBDA NIL :IN SB-DEBUG::FUNCALL-WITH-DEBUG-IO-SYNTAX) {1006DE117B}>)
resource_1    | 2: (SB-IMPL::%WITH-STANDARD-IO-SYNTAX #<CLOSURE (LAMBDA NIL :IN SB-DEBUG::FUNCALL-WITH-DEBUG-IO-SYNTAX) {1006DE114B}>)
resource_1    | 3: (PRINT-BACKTRACE :STREAM #<SB-IMPL::STRING-OUTPUT-STREAM {1006DE1083}> :START 0 :FROM :DEBUGGER-FRAME :COUNT 4611686018427387903 :PRINT-THREAD T :PRINT-FRAME-SOURCE NIL :METHOD-FRAME-STYLE NIL)
resource_1    | 4: (TRIVIAL-BACKTRACE:PRINT-BACKTRACE-TO-STREAM #<SB-IMPL::STRING-OUTPUT-STREAM {1006DE1083}>)
resource_1    | 5: (HUNCHENTOOT::GET-BACKTRACE)
resource_1    | 6: ((FLET #:HANDLER1222 :IN HUNCHENTOOT:HANDLE-REQUEST) #<MU-CL-RESOURCES::NO-SUCH-LINK {1006DCD193}>)
resource_1    | 7: (SIGNAL #<MU-CL-RESOURCES::NO-SUCH-LINK {1006DCD193}>)
resource_1    | 8: (ERROR MU-CL-RESOURCES::NO-SUCH-LINK :RESOURCE #<MU-CL-RESOURCES::RESOURCE {100B0400B3}> :PATH "prefLabel")
resource_1    | 9: ((:METHOD MU-CL-RESOURCES::FIND-RESOURCE-LINK-BY-PATH (MU-CL-RESOURCES::RESOURCE T)) #<MU-CL-RESOURCES::RESOURCE {100B0400B3}> "prefLabel") [fast-method]
resource_1    | 10: (MU-CL-RESOURCES::PROPERTY-PATH-FOR-FILTER-COMPONENTS #<MU-CL-RESOURCES::RESOURCE {100B0400B3}> ("prefLabel"))
resource_1    | 11: (MU-CL-RESOURCES::SPARQL-PATTERN-FILTER-STRING #<MU-CL-RESOURCES::RESOURCE {100B0400B3}> ?s :COMPONENTS ("prefLabel") :SEARCH "a")
resource_1    | 12: (MU-CL-RESOURCES::FILTER-BODY-FOR-SEARCH :RESOURCE #<MU-CL-RESOURCES::RESOURCE {100B0400B3}> :SOURCE-VARIABLE ?s :SPARQL-BODY "?s mu:uuid ?uuid; a tfdcat:Theme. ")
resource_1    | 13: ((:METHOD MU-CL-RESOURCES::LIST-CALL (MU-CL-RESOURCES::RESOURCE)) #<MU-CL-RESOURCES::RESOURCE {100B0400B3}>) [fast-method]
resource_1    | 14: ((LAMBDA (MU-CL-RESOURCES::BASE-PATH) :IN "/app/framework/call-specification.lisp") "themes")
resource_1    | 15: ((LAMBDA NIL :IN MU-SUPPORT::MK-RESPONSE-METHOD))
resource_1    | 16: ((:METHOD HUNCHENTOOT:HANDLE-REQUEST (HUNCHENTOOT:ACCEPTOR HUNCHENTOOT:REQUEST)) #<HUNCHENTOOT:EASY-ACCEPTOR (host *, port 80)> #<HUNCHENTOOT:REQUEST {1006CE2C93}>) [fast-method]
resource_1    | 17: ((:METHOD HUNCHENTOOT:PROCESS-REQUEST (T)) #<HUNCHENTOOT:REQUEST {1006CE2C93}>) [fast-method]
resource_1    | 18: (HUNCHENTOOT::DO-WITH-ACCEPTOR-REQUEST-COUNT-INCREMENTED #<HUNCHENTOOT:EASY-ACCEPTOR (host *, port 80)> #<CLOSURE (LAMBDA NIL :IN HUNCHENTOOT:PROCESS-CONNECTION) {1006CE201B}>)
resource_1    | 19: ((:METHOD HUNCHENTOOT:PROCESS-CONNECTION (HUNCHENTOOT:ACCEPTOR T)) #<HUNCHENTOOT:EASY-ACCEPTOR (host *, port 80)> #<USOCKET:STREAM-USOCKET {10069AABE3}>) [fast-method]
resource_1    | 20: ((:METHOD HUNCHENTOOT:PROCESS-CONNECTION :AROUND (HUNCHENTOOT:ACCEPTOR T)) #<HUNCHENTOOT:EASY-ACCEPTOR (host *, port 80)> #<USOCKET:STREAM-USOCKET {10069AABE3}>) [fast-method]
resource_1    | 21: ((FLET HUNCHENTOOT::PROCESS-CONNECTION% :IN HUNCHENTOOT::HANDLE-INCOMING-CONNECTION%) #<HUNCHENTOOT:EASY-ACCEPTOR (host *, port 80)> #<USOCKET:STREAM-USOCKET {10069AABE3}>)
resource_1    | 22: ((LAMBDA NIL :IN BORDEAUX-THREADS::BINDING-DEFAULT-SPECIALS))
resource_1    | 23: ((FLET #:WITHOUT-INTERRUPTS-BODY-1167 :IN SB-THREAD::INITIAL-THREAD-FUNCTION-TRAMPOLINE))
resource_1    | 24: ((FLET SB-THREAD::WITH-MUTEX-THUNK :IN SB-THREAD::INITIAL-THREAD-FUNCTION-TRAMPOLINE))
resource_1    | 25: ((FLET #:WITHOUT-INTERRUPTS-BODY-618 :IN SB-THREAD::CALL-WITH-MUTEX))
resource_1    | 26: (SB-THREAD::CALL-WITH-MUTEX #<CLOSURE (FLET SB-THREAD::WITH-MUTEX-THUNK :IN SB-THREAD::INITIAL-THREAD-FUNCTION-TRAMPOLINE) {7FFFF0D36C5B}> #<SB-THREAD:MUTEX "thread result lock" owner: #<SB-THREAD:THREAD "hunchentoot-worker-172.18.0.4:52220" RUNNING {10069ABC93}>> NIL T NIL)
resource_1    | 27: (SB-THREAD::INITIAL-THREAD-FUNCTION-TRAMPOLINE #<SB-THREAD:THREAD "hunchentoot-worker-172.18.0.4:52220" RUNNING {10069ABC93}> NIL #<CLOSURE (LAMBDA NIL :IN BORDEAUX-THREADS::BINDING-DEFAULT-SPECIALS) {10069ABC1B}> (#<SB-THREAD:THREAD "hunchentoot-worker-172.18.0.4:52220" RUNNING {10069ABC93}> #<SB-THREAD:THREAD "Swank 4005" RUNNING {1008E42463}> #<SB-THREAD:THREAD "Swank Sentinel" RUNNING {1008E420F3}> #<SB-THREAD:THREAD "hunchentoot-listener-*:80" RUNNING {100376C633}> #<SB-THREAD:THREAD "main thread" RUNNING {1002D57183}>) NIL NIL NIL NIL)
resource_1    | 28: ("foreign function: call_into_lisp")
resource_1    | 29: ("foreign function: new_thread_trampoline")
```